src/svd.jl

```diff
@@ -245,6 +245,24 @@ svdvals(A::AbstractVector{T}) where {T} = [convert(eigtype(T), norm(A))]
 svdvals(x::Number) = abs(x)
 svdvals(S::SVD{<:Any,T}) where {T} = (S.S)::Vector{T}
 
+"""
+    rank(S::SVD{<:Any, T}; atol::Real=0, rtol::Real=min(n,m)*ϵ) where {T}
+
+Compute the numerical rank of given an SVD object by counting how many singular values are greater 
```

Comment:
```suggestion
Compute the numerical rank of an SVD object by counting how many singular values are greater 
```
? Or "of `S`"? Or "of the `SVD` object `S`"?

Reply:
thanks!